### ISSUE TITLE
Update UI for summary page

### ISSUE THREAD

**Issue by SirusCodes** [OurTechCommunity/catchup#113]
Currently the summary page looks like this:

![image](https://user-images.githubusercontent.com/50910066/201114625-22281df2-2377-4f3c-9ae2-57b921e847f5.png)

To something that looks like this:

![image](https://user-images.githubusercontent.com/50910066/201115257-009d5779-2ffb-45b9-95ee-930db717ce33.png)

## What has changed? and Motivation

- Removed version number (Do we need it?)
- Removed about catchup section as it was taking up a large space and providing not much value to the reader who already know what catchups are. (We can add a one liner or add it somewhere else)
- Removed duplicate title also worked in #101 
- If we are generating OG images as screenshot of summary then I feel it's better to have more content upfront than these things.

What are your views @OurTechCommunity/core

**Comment by PranavDani** [comment]
![image](https://user-images.githubusercontent.com/56108484/201331114-31fd22bd-91b6-4d48-a6fc-51e1fc833f3f.png)

![image](https://user-images.githubusercontent.com/56108484/201330782-d4ded916-dc10-401c-b3c7-15c87026b732.png)

Currently, About CatchUp acts as the only link between [summaries](https://catchup.ourtech.community/summary), the latest summary, and [OTC CatchUp](https://catchup.ourtech.community). 
So what I suggest is, instead of having the hamburger on the top right, we can move it to the top left across every single page so that it does not collide with the dark/light switch on summaries (This can also allow the addition of the switch across all the pages.). (Also having it on the right side feels **RIGHT** for some reason.)


- This will allow users to use the hamburger across all the pages.
- Maintains consistency across all the pages.
- Allows a proper replacement for the above-mentioned changes.

**Comment by SirusCodes** [comment]
@PranavDani that does makes sense. But it should a whole another issue instead of being part of this issue.

**Comment by HarshKapadia2** [comment]
> - Removed about catchup section as it was taking up a large space and providing not much value to the reader who already know what catchups are. (We can add a one liner or add it somewhere else)

I think we really need at least one line explaining what OTC CatchUp sessions are. Although the probability of someone landing on a CatchUp summary page without knowing about CatchUp is low, we cannot just assume these things. (There are a lot of times when we share links with newcomers.) We can skip a section tag and just add a one line description and some links.

> - If we are generating OG images as screenshot of summary then I feel it's better to have more content upfront than these things.

We haven't decided anything about #3, so we can comment on this motivation only after that is decided.

**Comment by tusharnankani** [comment]
> I think we really need at least one line explaining what OTC CatchUp sessions are. Although the probability of someone landing on a CatchUp summary page without knowing about CatchUp is low, we cannot just assume these things. (There are a lot of times when we share links with newcomers.) We can skip a section tag and just add a one line description and some links.

Yes, sounds good.

@SirusCodes, would you like to work on this?

**Comment by SirusCodes** [comment]
@HarshKapadia2 
> I think we really need at least one line explaining what OTC CatchUp sessions are. Although the probability of someone landing on a CatchUp summary page without knowing about CatchUp is low, we cannot just assume these things. (There are a lot of times when we share links with newcomers.) We can skip a section tag and just add a one line description and some links.

Yeah makes sense, just a single line. But need to remove that big information box.

I would make a PR for this soon.

**Comment by tusharnankani** [comment]
Updates?

**Comment by SirusCodes** [comment]
I will try to do it tomorrow, thanks for reminding @tusharnankani
